gfx/drivers/vulkan.c

```diff
@@ -832,7 +835,7 @@ static bool vulkan_init_filter_chain(vk_t *vk)
 
 static void vulkan_init_resources(vk_t *vk)
 {
-   if (!vk)
+   if (!vk->context)
```

Comment:
Should this check `!vk` too?

Reply:
A bit too pedantic I think. This function is only called from two existing functions right now and in both cases, vk should already be a valid handle by the time it reaches this function call.